alchemiscale/interface/api.py

```diff
@@ -614,9 +614,8 @@ def get_network_actioned_tasks(
     validate_scopes(network_sk.scope, token)
 
     try:
-        # raises ValueError when a ScopedKey doesn't correspond to an AlchemicalNetwork
         taskhub_sk = n4js.get_taskhub(network_sk)
-    except ValueError as e:
+    except Exception as e:
```

Comment:
@dotsdl should we instead `except (ValueError, KeyError)` given your change?

Reply:
Ah good idea! This is better yes!